include/open62541cpp/open62541client.h

```diff
@@ -241,14 +241,24 @@ class Client
 
     /*!
      * \brief Initialise custom client config
+     * \param clientConfig client configuration
+     * \param setDefaultConfig flag to reset client config context
      */
-    void setCustomConfig(UA_ClientConfig* clientConfig)
+    void setCustomConfig(UA_ClientConfig* clientConfig, bool setDefaultConfig = true)
```

Comment:
This is the only change that matters. It is non-destructive so merging it should not break anything.